src/services/storage/drivers/postgres/index.ts

```diff
@@ -138,19 +171,16 @@ export class PostgresStorage implements StorageDriver {
 			return storedUrl
 		}
 
-		public async incVisitCount(id: string): Promise<void> {
-			await this.storage.db
-				.table<UrlInformation>('url_information')
-				.where('url_id', '=', id)
-				.increment('url_visit_count', 1)
+		public incVisitCount(id: string): void {
+			const currIds = Object.keys(this.storage.countersCache.visitCount)
+			if (currIds.includes(id)) this.storage.countersCache.visitCount[id] += 1
+			else this.storage.countersCache.visitCount[id] = 1
 		}
 
-		public async incInfoCount(id: string): Promise<void> {
-			await this.storage.db
-				.table<UrlInformation>('url_information')
-				.where('url_id', '=', id)
-				.increment('info_visit_count', 1)
-				.update({ lastUsed: new Date().toISOString() })
+		public incInfoCount(id: string): void {
+			const currIds = Object.keys(this.storage.countersCache.infoCount)
+			if (currIds.includes(id)) this.storage.countersCache.infoCount[id] += 1
+			else this.storage.countersCache.infoCount[id] = 1
```

Comment:
Same for `incVisitCount`.

```suggestion
			const infoCountCache = this.storage.countersCache.infoCount

			if (infoCountCache[id]) infoCountCache[id] += 1
			else infoCountCache[id] = 1
```